docs/03-02-examples.md

```diff
@@ -45,6 +45,42 @@ To build an image from your local sources and install Kyma based on this image,
 kyma install -s local --custom-image {IMAGE}
 ```
 
```

Comment:
That is a veryy good starting point and is what we need. Please extend it more with other examples regarding the --source flag and custom domains